cli/src/commands/explore/pallets/storage.rs

```diff
@@ -0,0 +1,214 @@
+use clap::Args;
+use color_eyre::{
+    eyre::{bail, eyre},
+    owo_colors::OwoColorize,
+};
+use indoc::{formatdoc, writedoc};
+use scale_typegen_description::type_description;
+use scale_value::Value;
+use std::fmt::Write;
+use std::write;
+
+use subxt::{
+    ext::scale_encode::EncodeAsType,
+    metadata::{
+        types::{PalletMetadata, StorageEntryType, StorageMetadata},
+        Metadata,
+    },
+};
+
+use crate::utils::{
+    create_client, first_paragraph_of_docs, parse_string_into_scale_value, type_example, FileOrUrl,
+    Indent, SyntaxHighlight,
+};
+
+#[derive(Debug, Clone, Args)]
+pub struct StorageSubcommand {
+    storage_entry: Option<String>,
+    #[clap(required = false)]
+    trailing_args: Vec<String>,
+}
+
+pub async fn explore_storage(
```

Comment:
With runtime APIs, we have an -e flag to actually try to execute a runtime API call. I feel like we should do the same for executing a storage request, so that by default we don't try to make a request but we can if we want. What do you think?

Reply:
Okay, I added an `-e` flag for storage entries as well and made the API very similar to the runtime apis.